base/src/main/java/io/spine/code/java/Annotations.java

```diff
@@ -51,10 +56,20 @@ private Annotations() {
     /**
      * Generates {@code \@Generated("by Spine Model Compiler")} annotation spec.
      *
-     * @return an {@link AnnotationSpec} describing the {@link Generated javax.annotation.Generated}
+     * @return an {@code AnnotationSpec} describing the {@link Generated javax.annotation.Generated}
      *         annotation
      */
     public static AnnotationSpec generatedBySpineModelCompiler() {
         return GENERATED;
     }
+
+    /**
+     * Generates {@code \@CanIgnoreReturnValue} annotation spec.
```

Comment:
I know another  approach is to use `{@literal @}`
```
Generates {@code {@literal @}CanIgnoreReturnValue} annotation spec.
```
Please investigate which one is better.

Reply:
It seems that just `{@code @CanIgnoreReturnValue}` is displayed okay in both Idea doc reference and the doc HTML generated by the Gradle.

When using any of the approaches for escaping the literal though, the doc is not displayed properly, so I just removed the escape mechanism here and in the similar case.